Code/Editor/Controls/SplineCtrlEx.cpp

```diff
@@ -2667,7 +2667,7 @@ void AbstractSplineWidget::SelectAll()
 }
 
 //////////////////////////////////////////////////////////////////////////
-void SplineWidget::SendNotifyEvent(int nEvent)
+void SplineWidget::SendNotifyEvent(unsigned int nEvent)
```

Comment:
It looks like the `SPLN_*` values are all unsigned 32 bit constants. Suggest that you use `uint32_t` instead (even const)
(same for the rest of this PR)
```suggestion
void SplineWidget::SendNotifyEvent(const uint32_t)
```

Reply:
`uint32_t` makes sense here. I changed `unsigned int` to `const uint32_t`